linkerd/manifest.json

```diff
@@ -2,7 +2,11 @@
   "maintainer": "help@datadoghq.com",
   "manifest_version": "1.0.0",
   "name": "linkerd",
+  "display_name": "Linkerd",
   "short_description": "Monitor your services health with metrics from linkerd.",
+  "metric_prefix": "linkerd",
```

Comment:
I believe there [needs to be a `.` at the end](https://github.com/DataDog/devops/wiki/Declarative-Integrations#example-metadata-json-fields-in-manifestjson) of `linkered`. So it would be:

```
"metric_prefix": "linkerd.",
```